docs_src/src/pages/documentation/example_app/templates.mdx

```diff
@@ -65,6 +65,16 @@ This is Robyn introduced him to the concept of routers and views.
   />
 </div>
 
+## Jinja Templating with `url_for`
 
+To integrate resource loading functionality into Jinja templates similar to Flask's `url_for`, follow these steps:
 
+**Loading Resources in Templates**: In your Jinja templates, use the `url_for` function to generate URLs for static resources.
 
+    ```html
+    <img src="{{ url_for('static', filename='images/planets.jpeg') }}" alt="Planets">
```

Comment:
@carrycooldude , why not use this example for testing itself?

Reply:
That's why earlier I added a static css file for doing the same for test but you said like don't add the css but do with html files that's why I didn't go with that...